lib/breakers/dashboard/static/app.css

```diff
@@ -0,0 +1,29 @@
+body {
```

Comment:
I think it would be nice if these files were maybe in a `app/views` folder and only added in as part of a Rails engine similar to sidekiq does this type of thing. Sidekiq actually uses `web` instead of `app`, but other libraries I've used have `app` and the benefit is, that later if you want to give the ability to customize views, it's just a matter of turning static assets into ones that are within the repo itself. You can have a custom rake set of rake tasks to generate and manage these views within your app.